package.json

```diff
@@ -29,11 +29,12 @@
     "clang-format": "1.2.2",
     "diff-match-patch": "^1.0.1",
     "glob": "7.1.2",
-    "google-closure-compiler": "20161024.3.0",
+    "google-closure-compiler": "20180610.0.2",
```

Comment:
Why do we need to change this?

Reply:
It's in the commit message, CC from 2016 would fail on the newer output that included more accurate cast expressions.